xla/service/cpu/onednn_config.proto

```diff
@@ -55,7 +55,7 @@ message OneDnnFusionConfig {
   repeated FusionKind ops = 1;
   // To avoid protobuf failures for specific decimal values,
   // the original float value alpha is type-casted to int32.
-  int32 alpha_typecast = 2;
+  repeated int32 alpha_typecast = 2;
```

Comment:
This will break protobuf backward compatibility, but it should be okay since we only use the proto in XLA's internal C++ implementation.